frontend/src/ui/strings.json

```diff
@@ -298,6 +300,7 @@
     "collectives": "colectivos",
     "comission": "Comisión",
     "contributor": "colaborando",
+    "coreContributors": "Colaborando",
     "coreContributors": "Colaboradores",
```

Comment:
the correct way of saying it is line 304 "coreContributors":"Colaboradores" 
also "coreContributor":"Colaborador"